integrationTests/node/heartbeat/heartbeat_test.go

```diff
@@ -12,9 +12,11 @@ import (
 	"github.com/ElrondNetwork/elrond-go/crypto/signing"
 	"github.com/ElrondNetwork/elrond-go/crypto/signing/mcl"
 	mclsig "github.com/ElrondNetwork/elrond-go/crypto/signing/mcl/singlesig"
+	"github.com/ElrondNetwork/elrond-go/heartbeat/data"
+	mock2 "github.com/ElrondNetwork/elrond-go/heartbeat/mock"
```

Comment:
rename mock 2 - should use only mocks from integration tests

Reply:
Crooked sons of bitches. Give us our money back.
 BTC dump ERD dump BTC pump ERD dump BTC stable ERD dump. Don'T buy ERD

Reply:
done